docs/HowTo/Get-Started/Start-Web3Signer.md

```diff
@@ -8,16 +8,16 @@ description: Getting started with Web3Signer
 
 * [Signing key configuration files] to access the required signing keys.
 
-Web3Signer supports the Ethereum 1, Ethereum 2, and Filecoin platforms, so you must specify the
+Web3Signer supports the execution layer (formerly Ethereum 1.0), consensus layer (Ethereum 2.0), and Filecoin platforms, so you must specify the
 signing mode, and the location of the signing key configuration files when starting Web3Signer.
 
-=== "Ethereum 1.0"
+=== "Execution layer"
```

Comment:
I wonder if "Execution layer clients" make more sense in the context? I'm not 100% on this.